Pipfile

```diff
@@ -35,6 +35,7 @@ typing-extensions = "==4.2.0"
 types-pyyaml = "==6.0.7"
 types-requests = "==2.27.20"
```

Comment:
Needs release notes and possibly a version bump if it hasn't been bumped already

Reply:
:+1: